runtime/common/BaseRemoteRESTQPU.h

```diff
@@ -939,7 +939,8 @@ class BaseRemoteRESTQPU : public cudaq::QPU {
       // Allow developer to disable remote sending (useful for debugging IR)
       if (getEnvBool("DISABLE_REMOTE_SEND", false))
         return;
-      future = executor->execute(codes, isObserve);
+      future = executor->execute(codes, isObserve, isRun,
```

Comment:
Below there is an `enum` defined. Why thread around a bag of `bool` arguments?

Reply:
Good point! Fixed in e73718937ce75439b0243e226e63153c9f4e6fc5